lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -271,14 +271,14 @@ function _campaign_resource_retrieve($nid) {
 /**
  * Retrieve and show index list response of campaigns requested.
  *
- * @param string $ids
- * @param bool $staff_pick
- * @param bool $mobile_app
- * @param string $mobile_app_date
- * @param string $term_ids
- * @param int $count
- * @param bool $random
- * @param int $page
+ * @param  string  $ids
```

Comment:
While I do find it more appealing, I would rather figure out something that everyone feels comfortable doing so we're all on the same page. Not sure what the proper venue for discussion is, but definitely something to tackle during documentation days...